<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerformInstallation</class>
 <widget class="QWidget" name="PerformInstallation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>323</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>485</width>
    <height>323</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>485</width>
    <height>323</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelIco">
   <property name="geometry">
    <rect>
     <x>441</x>
     <y>10</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelSelectInstall">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>7</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelSelectDir">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>26</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>99</y>
     <width>450</width>
     <height>164</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>450</width>
     <height>164</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>450</width>
     <height>164</height>
    </size>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>74</y>
     <width>480</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>480</width>
     <height>18</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>480</width>
     <height>18</height>
    </size>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="format">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNext">
   <property name="geometry">
    <rect>
     <x>313</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>398</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLastStep">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelProgress">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>57</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
